src/models/ui.ts

```diff
@@ -1,19 +1,21 @@
 import { types } from "mobx-state-tree";
 
+export const SpaceTypeEnum = types.enumeration("type", ["populations", "breeding", "organism", "dna", "none"]);
+export type SpaceType = typeof SpaceTypeEnum.Type;
 export type Space = "populations" | "breeding" | "organism" | "dna" | "none";
 
 export const UIModel = types
   .model("UI", {
     showPopulationGraph: false,
-    investigationPanelSpace: "none",
+    investigationPanelSpace: SpaceTypeEnum,
```

Comment:
I think you might be able to do something like `"none" as SpaceTypeEnum` here if you wanted

Reply:
Oh, that's nice too!